docs/app/documentation/component-docs/localization-editor/localization-editor-docs.component.html

```diff
@@ -0,0 +1,56 @@
+<h2>Localization Editor</h2>
+
+<description>
+    Basic usage of Localization Editor, to add some text on add-on there is <code>[label]</code> input.
```

Comment:
the 'to add some text on add-on' part is a bit confusing. Maybe 'Basic usage of the localization editor. The add-on content on the right-side can be customized through the <code>[label]</code> input.